### PR TITLE
tests for issue 1132

### PR DESCRIPTION
**Pull request by Knight219**
Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Have read [How To Contribute](https://github.com/Turfjs/turf/blob/master/CONTRIBUTING.md#how-to-contribute).
- [ ] Run `npm test` at the sub modules where changes have occurred.
- [ ] Run `npm run lint` to ensure code style at the turf module level.

Submitting a new TurfJS Module.

- [ ] Overview description of proposed module.
- [ ] Include JSDocs with a basic example.
- [ ] Execute `./scripts/generate-readmes` to create `README.md`.
- [ ] Add yourself to **contributors** in `package.json` using "Full Name <@GitHub Username>".

### CODE REVIEW

File: packages/turf-intersect/test/in/issue-1132-line.geojson [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+          [
+            [
+              -8479843.24714559,
```
Comment (DenisCarriere) [inline packages/turf-intersect/test/in/issue-1132-line.geojson]: @Knight219 You're coordinates are not using Decimal Degrees.

Coordinates that are not using a WGS84 projection is not going to be supported by TurfJS or the GeoJSON specification.

Could you replace your test data using Decimal Degrees instead.

Comment (Knight219) [reply packages/turf-intersect/test/in/issue-1132-line.geojson]: Intresting:). We use turf 4.0.0 with this format and all works well.

Comment (DenisCarriere) [reply packages/turf-intersect/test/in/issue-1132-line.geojson]: Humm... 🤔 No clue then, right now the output of your test data is returning a `Null` geometry.

https://travis-ci.org/Turfjs/turf/builds/308889447#L3074-L3076

Regardless, we won't be supporting no-decimal degree coordinates in TurfJS.

Comment (Knight219) [reply packages/turf-intersect/test/in/issue-1132-line.geojson]: I'll update our code and tests

Comment (Knight219) [reply packages/turf-intersect/test/in/issue-1132-line.geojson]: Looks that my data also in WGS84 with EPSG3857 but not EPSG4236

Comment (DenisCarriere) [reply packages/turf-intersect/test/in/issue-1132-line.geojson]: @Knight219 🤔 ? Does it also fail using WGS84? If so then send a commit with the change of test data.

If we don't come up with a solution we can skip it for now but at least the test case will be present for the future.

**Comment by DenisCarriere** [comment]
Not fixed, but the test case is now manually skipped in the module.

**Pull request merged**